### ISSUE TITLE
unicode subscripts not translating to Latex subscripts

### ISSUE THREAD

**Issue by isaacsas** [korsbo/Latexify.jl#216]
@korsbo any suggestions on how to fix this? 
```julia
rn = @reaction_network revbinding begin
               k₊, A + B --> C
               k₋, C --> A + B
       end k₊ k₋
repr(MIME"text/latex"(), rn)
```
gives
```
\\begin{align}\n\\require{mhchem}\n\\ce{ A + B &<=>[k\\_+][k\\_-] C}\n\\end{align}\n
```
while
```julia
latexify(rn, mathjax=false)
```
gives
```
L"\begin{align}
\ce{ A + B &<=>[$k\_n$][$k\_-$] C}
\end{align}
"
```
the problem being the `\_` and `\\_` instead of just `_` in the resulting equations.

This is on Latexify v0.15.13. I see a similar issue when converting these to ODEs.

**Comment by gustaphe** [comment]
This is (should be) fixed as of #215 . This is not released yet, but you can check out the development branch with `]dev Latexify`.

**Comment by isaacsas** [comment]
Ahh, perfect. Thanks for letting me know. Happy to wait on the next release, just wanted to make sure there wasn't some keyword arg I should be using too.
